Dockerfile

```diff
@@ -90,7 +92,7 @@ FROM inventree_base as prebuild
 
 ENV PATH=/root/.local/bin:$PATH
 RUN ./install_build_packages.sh --no-cache --virtual .build-deps && \
-    pip install --user -r base_requirements.txt -r requirements.txt --no-cache-dir && \
+    pip install --user uv --no-cache-dir && uv pip install -r base_requirements.txt -r requirements.txt --no-cache && \
```

Comment:
Is there another user now in the docker image called uv?

Reply:
No - see the cli docs for pip https://pip.pypa.io/en/stable/cli/pip_install/#cmdoption-user

Reply:
Oh you're right, I missed that this is already the next argument and not a value for the user argument